event-grid/subscribe-to-blob-storage/subscribe-to-blob-storage.ps1

```diff
@@ -4,19 +4,22 @@ $storageName = "contosostorage"
 # Provide an endpoint for handling the events.
 $myEndpoint = "<endpoint URL>"
 
+# Provide the name of the resource group to contain the storage account.
+$myResourceGroup="<resource group name>"
+
 # Create resource group
-New-AzureRmResourceGroup -Name myResourceGroup -Location westus2
+New-AzureRmResourceGroup -Name $myResourceGroup -Location westus2
```

Comment:
Be careful with `New-AzureRmResourceGroup`. If the user supplies an already-existing RG, the command will fail if there is a difference. In the case of testing these scripts, one used location `eastus2` to create the RG.

The script still runs and completes, but the user will see an error and may be confused.

Reply:
switching example scripts to all use westus2. Also, making resource group name user-supplied.